app/assets/javascripts/respondent_detail.es6

```diff
@@ -0,0 +1,31 @@
+$(document).ready(() => {
+  if ($("#bail_conditions_set").length) {
+    const bailConditionOptions = "input[type = radio][name = 'respondent[bail_conditions_set]']";
+    $(bailConditionOptions).on("change", setBailConditionsLabel);
+    setBailConditionsLabel();
+    if($('#bail_conditions_set_details-error').length) updateError();
+  };
+
+  function setBailConditionsLabel() {
+    const bailConditionsLabel = "label[for='bail_conditions_set_details']";
+    $(bailConditionsLabel).text($('#bail_conditions_set_details').attr(`data-bail-conditions-${selectedBailOption()}`));
```

Comment:
You should be able to use
```javascript
.data(`bail-conditions-${selectedBailOption()}`)
```
instead of
```javascript
.attr(`data-bail-conditions-${selectedBailOption()}`)
```